libs/resea/datetime.c

```diff
@@ -0,0 +1,81 @@
+#include <types.h>
+#include <resea/printf.h>
+#include <resea/datetime.h>
+
+bool is_leap_year(uint32_t year) {
```

Comment:
Nit: Define as `static` function. We don't use this function from other files for now.